execution/aggregate/hashaggregate.go

```diff
@@ -175,13 +176,13 @@ func (a *aggregate) Next(ctx context.Context) ([]model.StepVector, error) {
 }
 
 func (a *aggregate) aggregate(in []model.StepVector) error {
-	var warn warning
+	var err error
 	for i, vector := range in {
-		warn = coalesceWarn(warn, a.tables[i].aggregate(vector))
+		err = extwarnings.Coalesce(err, a.tables[i].aggregate(vector))
```

Comment:
We already have a warnings package, not sure if we need another one-file package.

Reply:
done